lib/block_view.go

```diff
@@ -3083,6 +3091,30 @@ func (bav *UtxoView) _connectUpdateGlobalParams(
 		}
 	}
 
+	if blockHeight >= bav.Params.ForkHeights.ProofOfStake2ConsensusCutoverBlockHeight {
```

Comment:
Note: we need to be able to configure this global param starting with the PoS setup block height
```suggestion
	if blockHeight >= bav.Params.ForkHeights.ProofOfStake1StateSetupBlockHeight {
```